tests/publish/ninjs_formatter_test.py

```diff
@@ -311,7 +311,7 @@ def test_item_with_usable_associations(self):
         formatted = json.loads(doc)
         self.assertIn('associations', formatted)
         self.assertIn('image', formatted['associations'])
-        image = formatted['associations']['image']
+        image = formatted['associations']['image'][0]
```

Comment:
not sure this works with the specs, could it be `formatted['associations']['image0']` instead?